client/webserver/site/src/html/orders.tmpl

```diff
@@ -35,12 +35,12 @@
         {{end}}
         <div class="apply-bttn d-hide"><button class="bg2 demi">[[[apply]]]</button></div>
       </div>
-      <div id="exportOrders" class="export-archived-records-bttn mt-3">
+      <div id="exportOrders" class="export-archived-records-bttn mt-3 pb-2">
```

Comment:
Will you please comment on all these frontend changes?  I didn't notice them in prior review, and I don't think they are relevant to the DMG package additions.

Reply:
The display didn't look right on webkit. See https://github.com/decred/dcrdex/pull/2333#issuecomment-1546698329.